src/jit/gentree.cpp

```diff
@@ -10196,7 +10196,7 @@ void Compiler::gtDispConst(GenTree* tree)
 #ifdef FEATURE_SIMD
                 if ((tree->gtFlags & GTF_ICON_SIMD_COUNT) != 0)
                 {
-                    printf(" Vector<T>.Count");
+                    printf(" Vector<T>, Vector64<T>, Vector128<T>, or Vector256<T>.Count");
```

Comment:
Rather than making this so verbose, I think it would be perfectly reasonable to just print "Vector count".

Reply:
Would you prefer `vector count` or `vector element count`?

The latter seems to be more explicit as to what the count is, but I don't feel particularly strongly about it 😄

Reply:
Vector element count sounds good.